docs/conf.py

```diff
@@ -116,7 +117,7 @@ def _git_version():
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
+# html_theme = 'default'
```

Comment:
wait, why change the html_theme? I think ReadTheDocs gets all ugly if it isn't default